org.civicrm.reqangular/src/common/angular-date/src/services/DateFactory.js

```diff
@@ -0,0 +1,20 @@
+var moment = require('../../../vendor/moment.min.js');
+
+function DateFactory(){
```

Comment:
In AMD module, it would be, indeed, better to use moment directly.
In CommonJS it prevents including moment multiple times.

Moreover, this class is supposed to do more things in the future.